src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioRuleSetTests.vb

```diff
@@ -137,7 +137,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
                 Dim fileChangeService = New MockVsFileChangeEx
                 Dim fileChangeWatcher = New FileChangeWatcher(Task.FromResult(Of IVsAsyncFileChangeEx)(fileChangeService))
 
-                Dim listenerProvider = workspace.ExportProvider.GetExportedValue(Of IAsynchronousOperationListenerProvider)
+                Dim listenerProvider = workspace.ExportProvider.GetExportedValue(Of AsynchronousOperationListenerProvider)
```

Comment:
❔ Why are we changing the ones that don't need access to the waiter functionality?

Reply:
They are, see line 154.

Reply:
(FWIW this change was made by deleting the extensions that were doing bad casting, and then fixing up tests that had errors)